docker/wiremock/stubs/mappings/retrievePatientInvalidNhsNumber.json

```
@@ -2,19 +2,19 @@
   "priority": 1,
   "request": {
     "method": "GET",
-    "url": "/GP0001/STU3/1/gpconnect/fhir/Patient?identifier=https://fhir.nhs.uk/Id/nhs-number%7CASDF"
+    "urlPattern": "\/.*\/STU3\/1\/gpconnect\/fhir\/Patient[?]identifier=https[:]\/\/fhir[.]nhs[.]uk\/Id\/nhs-number%7CASDF"
   },
   "response": {
     "status": 400,
     "bodyFileName": "patientInvalidNhsNumber.json",
     "headers": {
       "Server": "nginx",
       "Date": "{{now format='E, d MMM y HH:mm:ss z'}}",
-      "Content-Type": "application/fhir+json;charset=UTF-8",
+      "Content-Type": "application\/fhir+json;charset=UTF-8",
```

Did this actually fail or just applying the escaping across the board for consistency?
```suggestion
      "Content-Type": "application/fhir+json;charset=UTF-8",
```